spikeinterface/sortingcomponents/tests/test_peak_pipeline.py

```diff
@@ -62,17 +72,29 @@ def test_run_peak_pipeline():
                          **job_kwargs)
     
     # one step only : squeeze output
-    steps = [MyStep(recording, param0=6.6)]
-    step_one = run_peak_pipeline(recording, peaks, steps, job_kwargs, squeeze_output=True)
+    nodes = [
+        MyStep(recording, param0=6.6),
+    ]
+    step_one = run_peak_pipeline(recording, peaks, nodes, job_kwargs, squeeze_output=True)
     assert np.allclose(np.abs(peaks['amplitude']), step_one['abs_amplitude'])
     
-    # two step
-    steps = [MyStep(recording, param0=5.5), MyStepWithWaveforms(recording, ms_before=.5, ms_after=1.)]
-    step_one, step_two = run_peak_pipeline(recording, peaks, steps, job_kwargs)
-    assert np.allclose(np.abs(peaks['amplitude']), step_one['abs_amplitude'])
-    assert step_two.shape[0] == peaks.shape[0]
-    assert step_two.shape[1] == recording.get_num_channels()
-
+    # 3 nodes two have outputs
+    nodes = [
+        ExtractDenseWaveforms(recording, name='extract_waveforms', ms_before=.5, ms_after=1.,  have_global_output=False),
+        WaveformDenoiser(recording, name='denoiser', parents=['extract_waveforms'], have_global_output=False),
+        MyStep(recording, name='simple_step', param0=5.5),
+        MyStepWithWaveforms(recording, name='step_on_raw_wf', parents=['extract_waveforms']),
+        MyStepWithWaveforms(recording, name='step_on_denoised_wf', parents=['denoiser'])
+    ]
+
```

Comment:
In the example above, are you envisoning that steps that are not related to the other steps at all like the "simple_step"  will be used in the pipeline?

Reply:
Yes we can have node that not depend to each other. In short having parralel graph.